main.py

```diff
@@ -3,14 +3,86 @@
 import pdb
 import os
 
-def get_chapter_id( chapter_url ):
-  string_to_remove = "https://" if "https://" in chapter_url else "" 
-  substring = chapter_url.replace(string_to_remove+"mangadex.org/chapter/","")
-  id = substring[:-2]
+# sample of manga from mangadex : https://mangadex.org/title/31477/solo-leveling
+
+def get_manga_id( chapter_url ):
+  try:
+    id = int(''.join(filter(str.isdigit, chapter_url)))
+  except:
+    print('Please input correct link of manga')
+    exit(0)
   return id
```

Comment:
Rather than returning the id in the end of try except, can just return it in the `try` clause